package.json

```diff
@@ -21,10 +21,10 @@
     "test": "yarn link:emails; yarn jest",
     "test:integration": "yarn integration:servers:start \"yarn link:emails; yarn jest -c jest.integration.config.ts $TEST_FLAGS\"",
     "test:integration:watch": "yarn integration:servers:start \"yarn link:emails; yarn jest -c jest.integration.config.ts --watch\"",
-    "ci:test": "yarn integration:servers:start test",
-    "ci:test:integration": "yarn integration:servers:start \"yarn link:emails; yarn jest -c jest.integration.config.ts\"",
     "test:integration:cypress": "yarn integration:servers:start \"cd packages/cli && yarn cypress run -C cypressIntegration.config.ts\"",
     "test:integration:cypress:headed": "yarn integration:servers:start \"cd packages/cli && yarn cypress run -C cypressIntegration.config.ts --headed\"",
+    "ci:test": "yarn integration:servers:start test",
```

Comment:
does this need to run servers now?  can it instead do the same thing as the "test" command?

Reply:
it still needs to run servers, there's still a few other tests that don't pass without them.